### PR TITLE
Acceptance tests for CacheService

### LINKED ISSUE #1687: Add acceptance test for redis cache
### Problem

We have added redis cache class which takes care of shared caching, however we don't have any acceptance tests for it.

### Solution

Add acceptance tests according to the design doc: https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/docs/design/distributed-cache.md

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:
Adds a new test suite - `Acceptance Tests - Cache Service` which tests the integration of the `redis-server` running in a docker container from `hedera-local-node` with the relay class `CacheService`.

**Related issue(s)**:

Fixes #1687 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/index.spec.ts [hunk 1]
```diff
@@ -60,107 +60,112 @@ let startOperatorBalance: Hbar;
...
 
+  function loadTest(testFile) {
+    if (testFile !== 'index.spec.ts' && testFile.endsWith('.spec.ts')) {
+      require(`./${testFile}`);
+    }
+  }
+
+  function runLocalHederaNetwork() {
+    // set env variables for docker images until local-node is updated
+    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.41.0-alpha.3';
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/index.spec.ts:143]: New versions are available 
```suggestion
    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.42.0-alpha.2';
```
or
```suggestion
    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.41.0';
```

**Review by Nana-EC — APPROVED** [review]
LG, version upgrade suggestion if there's no impact to the tests

**Review by ebadiere — APPROVED** [review]
LG

**Review by Nana-EC — APPROVED** [review]

**Review by AlfredoG87 — APPROVED** [review]
LGTM

**Pull request merged**